tasks/libs/pipeline_notifications.py

```diff
@@ -147,20 +149,21 @@ def find_job_owners(failed_jobs, owners_file=".gitlab/JOBOWNERS"):
 
 
 def base_message(header, state):
-    return """{header} pipeline <{pipeline_url}|{pipeline_id}> for {commit_ref_name} {state}.
-{commit_title} (<{commit_url}|{commit_short_sha}>) by {author}""".format(  # noqa: FS002
-        header=header,
-        pipeline_url=os.getenv("CI_PIPELINE_URL"),
-        pipeline_id=os.getenv("CI_PIPELINE_ID"),
-        commit_ref_name=os.getenv("CI_COMMIT_REF_NAME"),
-        commit_title=os.getenv("CI_COMMIT_TITLE"),
-        commit_url="{project_url}/commit/{commit_sha}".format(  # noqa: FS002
-            project_url=os.getenv("CI_PROJECT_URL"), commit_sha=os.getenv("CI_COMMIT_SHA")
-        ),
-        commit_short_sha=os.getenv("CI_COMMIT_SHORT_SHA"),
-        author=get_git_author(),
-        state=state,
-    )
+    commit_title=os.getenv("CI_COMMIT_TITLE")
+    pipeline_url=os.getenv("CI_PIPELINE_URL")
+    pipeline_id=os.getenv("CI_PIPELINE_ID")
+    commit_ref_name=os.getenv("CI_COMMIT_REF_NAME")
+    commit_url_gitlab=f"{os.getenv('CI_PROJECT_URL')}/commit/{os.getenv('CI_COMMIT_SHA')}",
+    commit_url_github=f"{DATADOG_AGENT_GITHUB_REPO_URL}/commit/{os.getenv('CI_COMMIT_SHA')}",
+    commit_short_sha=os.getenv("CI_COMMIT_SHORT_SHA")
+    author=get_git_author()
+    parsed_pr_id_found = re.search(r'.*\(#([0-9]*)\)$', commit_title)
+    if parsed_pr_id_found:
+        parsed_pr_id = parsed_pr_id_found.group(1)
+    pr_url_github = f"{DATADOG_AGENT_GITHUB_REPO_URL}/pull/{parsed_pr_id}"
```

Comment:
:question: Is this only ever used for the `datadog-agent` repo? There are a few call sites that pass `datadog-agent` as header, which seems to indicate it might happen for other repos, but I wasn't able to find a reference to it so far

Reply:
That's a good point, double checking this

Reply:
Fixed with [5e72719](https://github.com/DataDog/datadog-agent/pull/18956/commits/5e72719a671d45dd2ba632e0900a97a549f96d56) ! PTAL